modules/virt-about-deploying-templates-custom-namespace.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-about-deploying-templates-custom-namespace.adoc
+
+[id="virt-about-deploying-templates-custom-namespace_{context}"]
+= About deploying templates to a custom namespace
+
+To add templates to a custom namespace, edit the `hco` custom resource (CR) and specify a custom namespace. Then, add the Red Hat templates to that custom namespace to enable the `ssp-operator` to populate the templates in the custom namespace.
```

Comment:
> Then, add the Red Hat templates to that custom namespace to enable the `ssp-operator` to populate the templates in the custom namespace.

 To add templates to a custom namespace, edit the `hco` custom resource (CR). Add `commonTemplatesNamespace` to the spec to specify a custom namespace. After HCO has been modified `ssp-operator` will populate the templates in the custom namespace.

Reply:
Updated the content.